README.md

```diff
@@ -11,6 +11,7 @@ expect from the core classes from the ECE and CS programs at CMU.
 - [18-240: Structure and Design of Digital Systems](ece_core/18240.md)
 - [18-290: Signals and Systems](ece_core/18290.md)
 - [18-500: ECE Design Experience](ece_core/18500.md)
+- [18-614: Microelectromechanical Systems (MEMS)](ece_core/18614.md)
```

Comment:
Hi, this is not a core class. Can you please add it to the electives section instead?

Reply:
For graduate students this course is listed as a core class as confirmed by my advisor. Is it only as it pertains to undergrad students?

Reply:
Core classes are classes that **everyone** has to take in order to graduate.

In M.S. you have a series of options for classes that contribute to your core. Sorry if there's a miscommunication here.

![image](https://user-images.githubusercontent.com/12106484/161490807-afd06f39-edb8-4952-b19d-f02d34c1d64d.png)

Reply:
I guess this repo is also geared towards CMU ECE B.S. (and CS), so it might not be as accurate for Master's Programs, which have a lot of variety / options to fulfill the requirements. If there are classes that every M.S. student has to take (I remember there were a few), feel free to start a section on that. I don't think 18-614 is one of them though.